### PR TITLE
 br: update doc for encrypt

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7404 and https://github.com/pingcap/docs-cn/pull/7428
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/use-br-command-line-tool.md [hunk 1]
```diff
@@ -481,9 +506,23 @@ br restore full -f 'mysql.usertable' -s $external_storage_url --ratelimit 128
 > - the system variable tables (`mysql.tidb`，`mysql.global_variables`) cannot be restored.
 > - the user information tables (such as `mysql.user` and `mysql.columns_priv`) cannot be restored.
 > - GC data cannot be restored.
-> 
+>
 > Restoring system tables might cause more compatibility issues. To avoid unexpected issues, **DO NOT** restore system tables in the production environment.
 
+### Decrypt backup data
+
+After the encrypted backup of the data, you need to pass in the corresponding decryption parameters to restore the data. If the decryption algorithm or key is incorrect, the data cannot be restored. You need to ensure that the decryption parameters and encryption parameters are consistent. An example of decrypting the backup data is as follows:
```
Comment (hfxsd) [inline br/use-br-command-line-tool.md:514]:
```suggestion
After encrypting the backup data, you need to pass in the corresponding decryption parameters to restore the data. You need to ensure that the decryption parameters and encryption parameters are consistent. If the decryption algorithm or key is incorrect, the data cannot be restored. 

The following is an example of decrypting the backup data:
```

Comment (hfxsd) [inline br/use-br-command-line-tool.md:514]:
```suggestion
After encrypting the backup data, you need to pass in the corresponding decryption parameters to restore the data. You need to ensure that the decryption parameters and encryption parameters are consistent. If the decryption algorithm or key is incorrect, the data cannot be restored. 

The following is an example of decrypting the backup data:
```

File: br/use-br-command-line-tool.md [hunk 2]
```diff
@@ -481,9 +506,25 @@ br restore full -f 'mysql.usertable' -s $external_storage_url --ratelimit 128
 > - the system variable tables (`mysql.tidb`，`mysql.global_variables`) cannot be restored.
 > - the user information tables (such as `mysql.user` and `mysql.columns_priv`) cannot be restored.
 > - GC data cannot be restored.
-> 
+>
 > Restoring system tables might cause more compatibility issues. To avoid unexpected issues, **DO NOT** restore system tables in the production environment.
 
+### Decrypt backup data
```
Comment (joccau) [inline br/use-br-command-line-tool.md:512]: what about the title of "Encrypt data during backup" and "Decrypt data during restore" int below context ?

File: br/use-br-command-line-tool.md [hunk 3]
```diff
@@ -246,6 +246,31 @@ LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/b
 
 In the above example, for the incremental backup data, BR records the data changes and the DDL operations during `(LAST_BACKUP_TS, current PD timestamp]`. When restoring data, BR first restores DDL operations and then the data.
 
+### Encrypt the backup data
```
Comment (joccau) [inline br/use-br-command-line-tool.md:249]:
```suggestion
### Encrypt data during backup
```

File: br/use-br-command-line-tool.md (hunk 2 above)
Comment (joccau) [inline br/use-br-command-line-tool.md:512]:
```suggestion
### Decrypt data during restore
```

**Comment by joccau** [comment]
LGTM

**Review by joccau — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**